src/assemble_workflow/bundle.py

```diff
@@ -68,10 +69,16 @@ def install_min(self) -> None:
         )
         self._execute(install_command)
 
-    def install_plugins(self) -> None:
-        for plugin in self.plugins:
-            logging.info(f"Installing {plugin.name}")
-            self.install_plugin(plugin)
+    def install_components(self) -> None:
+        for c in self.components.values():
+            if self.min_bundle == c:
+                pass
+            elif "plugins" in c.artifacts:
```

Comment:
Is `c` here a type of `BuildComponent`? or we iterating through the value of it.